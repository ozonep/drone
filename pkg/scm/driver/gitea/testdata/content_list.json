[{"name":"advanced.en-us.md","path":"docs/content/doc/advanced.en-us.md","sha":"de2bfeed6f2d3ea1d2f91e457e9450efc582bb31","type":"file","size":192,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/advanced.en-us.md?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/advanced.en-us.md","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/de2bfeed6f2d3ea1d2f91e457e9450efc582bb31","download_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/raw/branch/master/docs/content/doc/advanced.en-us.md","submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/advanced.en-us.md?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/de2bfeed6f2d3ea1d2f91e457e9450efc582bb31","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/advanced.en-us.md"}},{"name":"advanced.fr-fr.md","path":"docs/content/doc/advanced.fr-fr.md","sha":"04734a8f81f5641de2b6d738bc9b777c2cc541c6","type":"file","size":190,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/advanced.fr-fr.md?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/advanced.fr-fr.md","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/04734a8f81f5641de2b6d738bc9b777c2cc541c6","download_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/raw/branch/master/docs/content/doc/advanced.fr-fr.md","submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/advanced.fr-fr.md?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/04734a8f81f5641de2b6d738bc9b777c2cc541c6","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/advanced.fr-fr.md"}},{"name":"advanced.zh-cn.md","path":"docs/content/doc/advanced.zh-cn.md","sha":"1f7ebf81e723a0ba82a89eec18a2706ac1b0dace","type":"file","size":188,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/advanced.zh-cn.md?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/advanced.zh-cn.md","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/1f7ebf81e723a0ba82a89eec18a2706ac1b0dace","download_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/raw/branch/master/docs/content/doc/advanced.zh-cn.md","submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/advanced.zh-cn.md?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/1f7ebf81e723a0ba82a89eec18a2706ac1b0dace","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/advanced.zh-cn.md"}},{"name":"advanced","path":"docs/content/doc/advanced","sha":"8c43a4fb023735dfd026014bae92ad91414b88bd","type":"dir","size":0,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/advanced?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/advanced","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/8c43a4fb023735dfd026014bae92ad91414b88bd","download_url":null,"submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/advanced?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/8c43a4fb023735dfd026014bae92ad91414b88bd","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/advanced"}},{"name":"features.en-us.md","path":"docs/content/doc/features.en-us.md","sha":"9cc90b46a0d17864e5ce1749dbd1609a0921f8f7","type":"file","size":192,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/features.en-us.md?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/features.en-us.md","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/9cc90b46a0d17864e5ce1749dbd1609a0921f8f7","download_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/raw/branch/master/docs/content/doc/features.en-us.md","submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/features.en-us.md?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/9cc90b46a0d17864e5ce1749dbd1609a0921f8f7","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/features.en-us.md"}},{"name":"features.zh-cn.md","path":"docs/content/doc/features.zh-cn.md","sha":"68d2f71b11f6bb5b1ca31adad00bc0f6ed3e94d8","type":"file","size":188,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/features.zh-cn.md?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/features.zh-cn.md","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/68d2f71b11f6bb5b1ca31adad00bc0f6ed3e94d8","download_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/raw/branch/master/docs/content/doc/features.zh-cn.md","submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/features.zh-cn.md?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/68d2f71b11f6bb5b1ca31adad00bc0f6ed3e94d8","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/features.zh-cn.md"}},{"name":"features.zh-tw.md","path":"docs/content/doc/features.zh-tw.md","sha":"889e0df88213a937914ba43bb52956d0e54b1c98","type":"file","size":188,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/features.zh-tw.md?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/features.zh-tw.md","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/889e0df88213a937914ba43bb52956d0e54b1c98","download_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/raw/branch/master/docs/content/doc/features.zh-tw.md","submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/features.zh-tw.md?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/889e0df88213a937914ba43bb52956d0e54b1c98","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/features.zh-tw.md"}},{"name":"features","path":"docs/content/doc/features","sha":"e212486606cf756b3b626f0a5bb1dd386b83b698","type":"dir","size":0,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/features?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/features","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/e212486606cf756b3b626f0a5bb1dd386b83b698","download_url":null,"submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/features?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/e212486606cf756b3b626f0a5bb1dd386b83b698","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/features"}},{"name":"help.en-us.md","path":"docs/content/doc/help.en-us.md","sha":"5ad1dd7f1edb3b78a3e3fba2113ecd5b10625dc4","type":"file","size":176,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/help.en-us.md?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/help.en-us.md","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/5ad1dd7f1edb3b78a3e3fba2113ecd5b10625dc4","download_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/raw/branch/master/docs/content/doc/help.en-us.md","submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/help.en-us.md?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/5ad1dd7f1edb3b78a3e3fba2113ecd5b10625dc4","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/help.en-us.md"}},{"name":"help.zh-cn.md","path":"docs/content/doc/help.zh-cn.md","sha":"6af7aa1719b99e35874b76911aff29b9ad4bbc68","type":"file","size":180,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/help.zh-cn.md?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/help.zh-cn.md","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/6af7aa1719b99e35874b76911aff29b9ad4bbc68","download_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/raw/branch/master/docs/content/doc/help.zh-cn.md","submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/help.zh-cn.md?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/6af7aa1719b99e35874b76911aff29b9ad4bbc68","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/help.zh-cn.md"}},{"name":"help","path":"docs/content/doc/help","sha":"91adc879a877f6c44aab1f27453bfa2679e4c3a0","type":"dir","size":0,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/help?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/help","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/91adc879a877f6c44aab1f27453bfa2679e4c3a0","download_url":null,"submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/help?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/91adc879a877f6c44aab1f27453bfa2679e4c3a0","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/help"}},{"name":"installation.en-us.md","path":"docs/content/doc/installation.en-us.md","sha":"4257521d973b0d28a7040f9030f701bbd1bf7bef","type":"file","size":208,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/installation.en-us.md?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/installation.en-us.md","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/4257521d973b0d28a7040f9030f701bbd1bf7bef","download_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/raw/branch/master/docs/content/doc/installation.en-us.md","submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/installation.en-us.md?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/4257521d973b0d28a7040f9030f701bbd1bf7bef","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/installation.en-us.md"}},{"name":"installation.fr-fr.md","path":"docs/content/doc/installation.fr-fr.md","sha":"55b48bda3eca4e459c12e44d54de9c4573d9b0b7","type":"file","size":208,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/installation.fr-fr.md?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/installation.fr-fr.md","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/55b48bda3eca4e459c12e44d54de9c4573d9b0b7","download_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/raw/branch/master/docs/content/doc/installation.fr-fr.md","submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/installation.fr-fr.md?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/55b48bda3eca4e459c12e44d54de9c4573d9b0b7","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/installation.fr-fr.md"}},{"name":"installation.zh-cn.md","path":"docs/content/doc/installation.zh-cn.md","sha":"8f57e0f00c273619c1ce66f04f7536f62d830e88","type":"file","size":196,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/installation.zh-cn.md?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/installation.zh-cn.md","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/8f57e0f00c273619c1ce66f04f7536f62d830e88","download_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/raw/branch/master/docs/content/doc/installation.zh-cn.md","submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/installation.zh-cn.md?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/8f57e0f00c273619c1ce66f04f7536f62d830e88","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/installation.zh-cn.md"}},{"name":"installation.zh-tw.md","path":"docs/content/doc/installation.zh-tw.md","sha":"f955e994ac09834f546212a6ec494978a4d10039","type":"file","size":196,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/installation.zh-tw.md?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/installation.zh-tw.md","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/f955e994ac09834f546212a6ec494978a4d10039","download_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/raw/branch/master/docs/content/doc/installation.zh-tw.md","submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/installation.zh-tw.md?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/f955e994ac09834f546212a6ec494978a4d10039","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/installation.zh-tw.md"}},{"name":"installation","path":"docs/content/doc/installation","sha":"2a3ff66218ae980c07444b8c97b710b3d7b5b846","type":"dir","size":0,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/installation?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/installation","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/2a3ff66218ae980c07444b8c97b710b3d7b5b846","download_url":null,"submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/installation?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/2a3ff66218ae980c07444b8c97b710b3d7b5b846","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/installation"}},{"name":"upgrade.en-us.md","path":"docs/content/doc/upgrade.en-us.md","sha":"9623ff9f1eaf74e9ebc03f5869bc0b9d2b3372d3","type":"file","size":188,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/upgrade.en-us.md?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/upgrade.en-us.md","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/9623ff9f1eaf74e9ebc03f5869bc0b9d2b3372d3","download_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/raw/branch/master/docs/content/doc/upgrade.en-us.md","submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/upgrade.en-us.md?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/9623ff9f1eaf74e9ebc03f5869bc0b9d2b3372d3","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/upgrade.en-us.md"}},{"name":"upgrade.fr-fr.md","path":"docs/content/doc/upgrade.fr-fr.md","sha":"ca08daf9c096bf027ba95a13b44c5fe4d0a47afe","type":"file","size":198,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/upgrade.fr-fr.md?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/upgrade.fr-fr.md","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/ca08daf9c096bf027ba95a13b44c5fe4d0a47afe","download_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/raw/branch/master/docs/content/doc/upgrade.fr-fr.md","submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/upgrade.fr-fr.md?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/ca08daf9c096bf027ba95a13b44c5fe4d0a47afe","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/upgrade.fr-fr.md"}},{"name":"upgrade.zh-cn.md","path":"docs/content/doc/upgrade.zh-cn.md","sha":"d12e150b9cbce2c26acb64c724914cec8667bb5d","type":"file","size":186,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/upgrade.zh-cn.md?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/upgrade.zh-cn.md","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/d12e150b9cbce2c26acb64c724914cec8667bb5d","download_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/raw/branch/master/docs/content/doc/upgrade.zh-cn.md","submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/upgrade.zh-cn.md?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/d12e150b9cbce2c26acb64c724914cec8667bb5d","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/upgrade.zh-cn.md"}},{"name":"upgrade.zh-tw.md","path":"docs/content/doc/upgrade.zh-tw.md","sha":"b57fccbdd578e9c981171838a9dd1e152f69bce9","type":"file","size":186,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/upgrade.zh-tw.md?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/upgrade.zh-tw.md","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/b57fccbdd578e9c981171838a9dd1e152f69bce9","download_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/raw/branch/master/docs/content/doc/upgrade.zh-tw.md","submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/upgrade.zh-tw.md?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/b57fccbdd578e9c981171838a9dd1e152f69bce9","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/upgrade.zh-tw.md"}},{"name":"upgrade","path":"docs/content/doc/upgrade","sha":"ff064862ee6ca9aa7bfda908c5354c83fbac25a4","type":"dir","size":0,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/upgrade?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/upgrade","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/ff064862ee6ca9aa7bfda908c5354c83fbac25a4","download_url":null,"submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/upgrade?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/ff064862ee6ca9aa7bfda908c5354c83fbac25a4","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/upgrade"}},{"name":"usage.en-us.md","path":"docs/content/doc/usage.en-us.md","sha":"6be9769d64603f93e05e099a8b2dcbceae1e98df","type":"file","size":180,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/usage.en-us.md?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/usage.en-us.md","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/6be9769d64603f93e05e099a8b2dcbceae1e98df","download_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/raw/branch/master/docs/content/doc/usage.en-us.md","submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/usage.en-us.md?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/6be9769d64603f93e05e099a8b2dcbceae1e98df","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/usage.en-us.md"}},{"name":"usage.zh-cn.md","path":"docs/content/doc/usage.zh-cn.md","sha":"c533df662481ea381cba02f0cb7996a3c1b377fd","type":"file","size":194,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/usage.zh-cn.md?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/usage.zh-cn.md","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/c533df662481ea381cba02f0cb7996a3c1b377fd","download_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/raw/branch/master/docs/content/doc/usage.zh-cn.md","submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/usage.zh-cn.md?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/c533df662481ea381cba02f0cb7996a3c1b377fd","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/usage.zh-cn.md"}},{"name":"usage","path":"docs/content/doc/usage","sha":"ac7d00a3401c3299068e09da540badfaf29d5e58","type":"dir","size":0,"encoding":null,"content":null,"target":null,"url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/usage?ref=master","html_url":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/usage","git_url":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/ac7d00a3401c3299068e09da540badfaf29d5e58","download_url":null,"submodule_git_url":null,"_links":{"self":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/usage?ref=master","git":"https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/ac7d00a3401c3299068e09da540badfaf29d5e58","html":"https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/usage"}}]