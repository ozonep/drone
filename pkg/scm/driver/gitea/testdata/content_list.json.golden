[
  {
    "path": "docs/content/doc/advanced.en-us.md",
    "kind": "file"
  },
  {
    "path": "docs/content/doc/advanced.fr-fr.md",
    "kind": "file"
  },
  {
    "path": "docs/content/doc/advanced.zh-cn.md",
    "kind": "file"
  },
  {
    "path": "docs/content/doc/advanced",
    "kind": "directory"
  },
  {
    "path": "docs/content/doc/features.en-us.md",
    "kind": "file"
  },
  {
    "path": "docs/content/doc/features.zh-cn.md",
    "kind": "file"
  },
  {
    "path": "docs/content/doc/features.zh-tw.md",
    "kind": "file"
  },
  {
    "path": "docs/content/doc/features",
    "kind": "directory"
  },
  {
    "path": "docs/content/doc/help.en-us.md",
    "kind": "file"
  },
  {
    "path": "docs/content/doc/help.zh-cn.md",
    "kind": "file"
  },
  {
    "path": "docs/content/doc/help",
    "kind": "directory"
  },
  {
    "path": "docs/content/doc/installation.en-us.md",
    "kind": "file"
  },
  {
    "path": "docs/content/doc/installation.fr-fr.md",
    "kind": "file"
  },
  {
    "path": "docs/content/doc/installation.zh-cn.md",
    "kind": "file"
  },
  {
    "path": "docs/content/doc/installation.zh-tw.md",
    "kind": "file"
  },
  {
    "path": "docs/content/doc/installation",
    "kind": "directory"
  },
  {
    "path": "docs/content/doc/upgrade.en-us.md",
    "kind": "file"
  },
  {
    "path": "docs/content/doc/upgrade.fr-fr.md",
    "kind": "file"
  },
  {
    "path": "docs/content/doc/upgrade.zh-cn.md",
    "kind": "file"
  },
  {
    "path": "docs/content/doc/upgrade.zh-tw.md",
    "kind": "file"
  },
  {
    "path": "docs/content/doc/upgrade",
    "kind": "directory"
  },
  {
    "path": "docs/content/doc/usage.en-us.md",
    "kind": "file"
  },
  {
    "path": "docs/content/doc/usage.zh-cn.md",
    "kind": "file"
  },
  {
    "path": "docs/content/doc/usage",
    "kind": "directory"
  }
]
